_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
+
+After [Creating a custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/) you can contribute your workload for other OpenSearch Benchmark users to use. If you plan on configuring a workload, use the following guidelines.
+
+## README and license
+
+Each workload must include a detailed `README.MD` file, which should include the following:  
+
+- The purpose of the workload and what users should use the workload. When creating a description, think about what specific use case your workload address that other workloads included in OpenSearch Benchmark do not.
+- An overview of the data set use. Make sure that any data included in the data set does not contain any properietary data or PII. 
```

Comment:
- "the data set use" --> "the data set used" 
- "PII" or "Personal Identifiable Information (PII)" (we should spell it out and then abbreviate)